.github/workflows/template-only-ci-infra.yml

```diff
@@ -11,6 +11,7 @@ on:
       - template-only-test/**
       - .github/workflows/template-only-ci-infra.yml
       - app/Dockerfile
+  workflow_dispatch:
```

Comment:
@lorenyu  Did you mean to add this key with an empty value?

Reply:
yeah that's the main goal of this PR. the workflow_dispatch configuration allows this workflow to be triggered manually, similar to this one: https://github.com/navapbc/template-application-nextjs/blob/2f1fefbb9c45b78ffe6fb1b8a4d519926ce4152d/.github/workflows/cd-storybook.yml#L12